using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+# Using the Portaal taak Plugin
+
+The Portaal Taak plugin is used to generate "taak" (task) objects within the Objecten API for a BPMN 
+user task. As a result, such tasks can be completed via external applications, such as the NL Portal. This feature 
```

Comment:
NL Portal isn't something that has been explained yet. I think giving examples is good, I just don't know if it makes sense to mention the NL Portal, since the intention was to not document portal related stuff (obviously stuff that's in Valtimo is the exception to this).

Reply:
It's a tough one. I see your point, but i do think that it's the only actual example that we have right now. It makes the goal of this feature a bit more clear IMO. @ThomasRitense what do you think?